.gitignore

```diff
@@ -24,6 +24,7 @@ website/node_modules
 *.iml
 *.test
 *.iml
+.vscode
```

Comment:
I was not aware of the `settings.json` file added for VScode dev. So we will likely want to add a `!.vscode/settings.json` after this so it does not ignore that file

Reply:
Yeah, since yall had that in there is why I initially included the 2 other files that also provide IDE debugging - would you like me to add that back or just the settings.json?

Reply:
Ok I think lets just remove the `.vscode` for now and we can adjust it alter if needed. Thanks